CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 
 - New algorithms:
 
```

Comment:
```suggestion
```

Reply:
@huonw I think github hasn't rendered your suggestion properly! what do you mean?

Reply:
The rendering is correct, although definitely a bit misleading: remove the blank line, to match how we've done it before, e.g. 0.9.0 has:
```markdown
- New algorithms:
  - Cluster-GCN: an extension of GCN that can be trained using SGD, with demo [\#487](https://github.com/stellargraph/stellargraph/issues/487)
```

That one renders like:

![image](https://user-images.githubusercontent.com/1203825/75218386-43d3ac00-57ee-11ea-9599-7bc41f78b34b.png)

while this one has extraneous spaces (and I think it gets worse if there's more items in the list):

![image](https://user-images.githubusercontent.com/1203825/75218398-4c2be700-57ee-11ea-9d6e-2fc704c71f7b.png)